src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceDay = 40;
+  const rentPrice = days * priceDay;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  let total = rentPrice;
+  const LONG_TERM = 7;
+  const SMALL_TERM = 3;
+
+  if (days >= LONG_TERM) {
```

Comment:
Consider adding a comment here explaining why you are checking if days is greater or equal to LONG_TERM and why you are applying a discount in this case. It will improve code readability.